reference/api/bindings_api.md

```diff
@@ -80,6 +80,73 @@ def incoming():
     return "Kafka Event Processed!"
 ```
 
+### Binding subscription
```

Comment:
Binding subscription doesn't seem the correct term here. It has potential to confuse customers with pubsub.

Reply:
I addressed the comments from Aman. Take a look